functions/go/project-service-list/README.md

```diff
@@ -0,0 +1,117 @@
+# project-services-list
+
+## Overview
+
+<!--mdtogo:Short-->
+
+The `project-services-list` function generates [project service](https://cloud.google.com/config-connector/docs/reference/resource-docs/serviceusage/service)
+resources from a list of services to enable [GCP APIs](https://cloud.google.com/apis) within a specified project.
```

Comment:
May be I am missing something, but `enable-gcpservice-apis` might be better name for this function. right ? 
I think there is another function (similar to this) also in progress.

Reply:
I tried to align with our Terraform equivalent https://github.com/terraform-google-modules/terraform-google-project-factory/tree/master/modules/project_services
/cc @morgante if you have any suggestions

Reply:
I think we could consider combining with https://github.com/GoogleContainerTools/kpt-functions-catalog/pull/554 and calling the combined function `enable-gcp-services`. By default, it would just process the ProjectServices CRD but if you set `generate: true` it could also generate new services.

@g-awmalik What do you think?

Reply:
I agree. @bharathkkb and I were on the fence about having another function that is kinda similar but not. This flag would address both requirements and eliminate an additional (almost redundant) function.

If all agree, I can work with @bharathkkb to merge the logic in here.

Reply:
Merging these 2 sounds reasonable to me, and it can help reduce user confusion.

Reply:
SGTM, we can the generate option in a followup PR.

Reply:
I'll close the [554](https://github.com/GoogleContainerTools/kpt-functions-catalog/pull/554) in that case and work with Bharath to to move that logic here.

Reply:
Great.